vault/README.md

```diff
@@ -12,6 +12,8 @@ The Vault check is included in the [Datadog Agent][3] package. No additional ins
 
 #### Prerequisites
 
+Ensure you have enabled [Prometheus metrics in the Vault configuration][16].
+
 For Vault check to work properly, you need to a) enable unauthenticated access to vault metrics or b) provide a Vault client token.
```

Comment:
Should we number these prereqs since there is more than 1.

Reply:
Yes, good idea ... Getting all the markdown indentation right is the challenge. :) Working on an updated commit for this.

Reply:
Updated.